csrc/quantization/fp8/common.cu

```diff
@@ -21,20 +28,26 @@ __device__ __forceinline__ float atomicMaxFloat(float* addr, float value) {
   return old;
 }
 
-#define FP8_E4M3_MAX std::numeric_limits<c10::Float8_e4m3fn>::max()
+#define FP8_E4M3_MAX std::numeric_limits<FP8_TYPE>::max()
```

Comment:
N.B. AMD uses up to `224` not `240`

Reply:
I suppose this should be handled by torch.

Reply:
+1 it would be best to leave this up to torch if possible

Reply:
It is bit too strong to ask torch to make the change, as our current HW does support up to 240, nothing blocks it is being used. here in inference stack to maintain Interop with OCP via manipulating scaling factors, we introduced scale-by-2, to assure that I prefer us stick to 224 (maps to OCP-MAX 448).

Not ask us to change real value of `numeric_limits<c10::Float8_e4m3fnuz>::max()`, just ask to have a redef here.

Reply:
Okay I see the justification now, that is fine with me. Please just leave a comment explaining the special definition